### ISSUE TITLE
Training on other datasets

### ISSUE THREAD

**Issue by Ianpengg** [haomo-ai/MotionSeg3D#20]
Hi thanks for sharing this great work

I'm wondering if it's possible to train on other datasets besides the ones that you've provided, such as the Oxford RobotCar dataset. 
it also provide the lidar point cloud data 
Would it be feasible to train on this dataset, and if so, what kind of information would you need from us to facilitate the training process?
ex: point level label , groundtruth pose ...etc? 
Thanks in advance for your help!

**Comment by MaxChanger** [comment]
Hi @Ianpengg, thank you for your interest in our project.
There's a bit of discussion here [#2](https://github.com/haomo-ai/MotionSeg3D/issues/2) and [LiDAR-MOS#52](https://github.com/PRBonn/LiDAR-MOS/issues/52#issuecomment-1174820247), and as long as your data can be converted into the same format as KITTI Odometry (or SemanticKITTI), it can be seamlessly integrated into the framework.

**Comment by Ianpengg** [comment]
Thanks for replies~ I plan to first test the dataset through inference to see if the pretrained model can work for my purposes. After that, I will manually label the moving objects in the LiDAR data. Do you know if there are any efficient labeling tools or modules available that can help me quickly label the entire sequence? I estimate that there are about 10,000 to 20,000 frames that need to be labeled. Thank you.

**Comment by MaxChanger** [comment]
Hi, I thinks there are some helpful links in the [kitti_road_mos.md](https://github.com/haomo-ai/MotionSeg3D/blob/master/config/kitti_road_mos.md).
> More specifically, we first use auto-mos labeling method ([link](https://arxiv.org/pdf/2201.04501.pdf)) to automatically generate the MOS labels for KITTI-Road data. We then use a point labeler ([link](https://github.com/jbehley/point_labeler)) to manually refined the labels.

**Comment by Ianpengg** [comment]
Thanks, this issue  can be closed~
